keps/sig-node/2902-cpumanager-distribute-cpus-policy-option/README.md

```diff
@@ -182,12 +209,33 @@ No changes. Existing container will not see their allocation changed. New contai
 
 - A specific e2e test will demonstrate that the default behaviour is preserved when the feature gate is disabled, or when the feature is not used (2 separate tests)
 
+### Rollout, Upgrade and Rollback Planning
+
+###### How can a rollout or rollback fail? Can it impact already running workloads?
+
+- A rollout or rollback can fail if the feature gate and the policy option are not configured properly and kubelet fails to start.
+
+###### What specific metrics should inform a rollback?
+
+As part of graduation of this feature, we plan to add metric `cpu_manager_numa_allocation_spread` to see how the CPUs are distributed across NUMA nodes.
+This can be used to see the CPU distribution across NUMA and will provide an indication of a rollback.
+
+###### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?
+
+Not Applicable.
```

Comment:
I think this is applicable. Could you do a manual test of this?

Reply:
The reason it is not applicable is because this policy option only affects pods that meet certain conditions and are scheduled after the upgrade. Running pods will be unaffected by any change. I will add more details in the KEP to capture this.

Reply:
I think just expanding the current text with the reasoning above is fine

Reply:
Done :)